src/plotman/plotman.py

```diff
@@ -94,8 +94,24 @@ def parse_args(self) -> typing.Any:
         p_analyze.add_argument('--bybitfield',
                 action='store_true',
                 help='slice by bitfield/non-bitfield sorting')
-        p_analyze.add_argument('logfile', type=str, nargs='+',
+        p_analyze.add_argument('--logfile', type=str, nargs='+', default=None,
                 help='logfile(s) to analyze')
+        p_analyze.add_argument('--logdir', type=str, default=None,
+                help='directory containing multiple logfiles to analyze')
```

Comment:
This and the same below in `graph` seem to be unused.  I think if this is removed then the `--` can be removed from `--logfile` above as well.